packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -609,6 +610,54 @@ export interface CallAdapterCallOperations {
    * @public
    */
   disposeStreamView(remoteUserId?: string, options?: VideoStreamOptions): Promise<void>;
+  /* @conditional-compile-remove(together-mode) */
+  /**
+   * Create the html view for a stream.
+   *
+   * @remarks
+   * This method is implemented for composite
+   *
+   * @param featureName - Name of feature to render
+   * @param options - Options to control how video streams are rendered {@link @azure/communication-calling#VideoStreamOptions }
+   *
+   * @beta
+   */
+  createTogetherModeStreamViews(options?: VideoStreamOptions): Promise<void | TogetherModeStreamViewResult>;
+  /* @conditional-compile-remove(together-mode) */
+  /**
+   * Start Together mode.
+   *
+   * @beta
+   */
+  startTogetherMode(): Promise<void>;
+  /* @conditional-compile-remove(together-mode) */
+  /**
+   * Recalculate the seating positions for together mode.
+   *
+   * @remarks
+   * This method is implemented for composite
+   *
+   * @param width - Width of the container
+   * @param height - Height of the container
+   *
+   * @beta
+   */
+  setTogetherModeSceneSize(width: number, height: number): void;
```

Comment:
Is this something that we would want to give contoso access to? I think that we could have this set internally so that our resize observers are setting this.

Reply:
For now I removed it and would look into it in my next PR